### ISSUE TITLE
Project "sales pitch"

### ISSUE THREAD

**Issue by jupart** [LibrePCB/LibrePCB#220]

**Comment by ubruhin** [comment]
You're right, it's still on my ToDo list to create a comparison between LibrePCB and other EDA tools (see also https://github.com/LibrePCB/LibrePCB.github.io/issues/1) :wink:

Btw, I will give an introduction to LibrePCB at FOSDEM, maybe you are interested in that: http://librepcb.org/blog/2018-01-02_librepcb_at_fosdem_18/

**Comment by jupart** [comment]
I'll definitely be checking that out (and keeping an eye on this project), thanks!

**Comment by ubruhin** [comment]
I think the new website should now give a much better overview about the advantages of LibrePCB, so let's close this issue for now. If there are still requests to add more information, please report it at the repository https://github.com/LibrePCB/librepcb-website.
